controllers/scriptStorage.js

```diff
@@ -733,7 +737,11 @@ exports.webhook = function (aReq, aRes) {
     payload.commits.forEach(function (aCommit) {
       aCommit.modified.forEach(function (aFilename) {
         if (aFilename.substr(-8) === '.user.js') {
-          repo[aFilename] = '/' + encodeURI(aFilename);
+          console.log([
+            'Webhook filename:',
+            '  ' + aFilename
+          ].join('\n')); // TODO: After some data collected, reaffirm and remove this
+          repo[aFilename] = '/' + encodeURI(aFilename); // NOTE: Watchpoint
```

Comment:
Some data here:

`aFilename` : [`src/RFC 2606§3/RFC 2606§3 - ä %, Unit Test/RFC 2606§3 - ä %, Unit Test.user.js`](https://github.com/Martii/UserScripts/blob/master/src/RFC%202606%C2%A73/RFC%202606%C2%A73%20-%20%C3%A4%20%25%2C%20Unit%20Test/RFC%202606%C2%A73%20-%20%C3%A4%20%25%2C%20Unit%20Test.user.js) on trigger.

Refs:
1. [`0d06785`/libs/repoManager.js#L119-L120](https://github.com/OpenUserJs/OpenUserJS.org/blob/0d067859c2a49bcb971a2838111e31bde3d0de5b/libs/repoManager.js#L119-L120)
2. [https://raw.githubusercontent.com/Martii/UserScripts/master/src/RFC%202606%C2%A73/RFC%202606%C2%A73%20-%20%C3%A4%20%25%2C%20Unit%20Test/RFC%202606%C2%A73%20-%20%C3%A4%20%25%2C%20Unit%20Test.user.js](https://raw.githubusercontent.com/Martii/UserScripts/master/src/RFC%202606%C2%A73/RFC%202606%C2%A73%20-%20%C3%A4%20%25%2C%20Unit%20Test/RFC%202606%C2%A73%20-%20%C3%A4%20%25%2C%20Unit%20Test.user.js) Raw URL copied from address bar
3. `encodeURI('RFC 2606§3 - ä %, Unit Test.user.js')` := `RFC%202606%C2%A73%20-%20%C3%A4%20%25,%20Unit%20Test.user.js`
4. `encodeURIComponent('RFC 2606§3 - ä %, Unit Test.user.js')` := `RFC%202606%C2%A73%20-%20%C3%A4%20%25%2C%20Unit%20Test.user.js`

@Zren Cc: @sizzlemctwizzle 
Would either of you say there is an issue here? *(look for the comma... e.g. `%2C` and `,` ... in list item 2, 3 and 4)*

Everything appears alright [on pro](https://openuserjs.org/scripts/Marti/RFC_2606%C2%A73_-_%C3%A4_%25%2C_Unit_Test) with the JSON... but I've also picked simple values in the Userscript and something still doesn't feel right.